pallets/msa/src/replay_tests.rs

```diff
@@ -35,17 +35,17 @@ pub fn user_adds_key_to_msa(
 	delegator_pair: sp_core::sr25519::Pair,
 	new_pair: sp_core::sr25519::Pair,
 ) {
```

Comment:
I thought there was a new replay test for the other signature, does there need to be one?

Reply:
You mean this: https://github.com/LibertyDSNP/frequency/pull/617/files#diff-376fd47e95f7a80f67ffe3cf86e50bfd6d4d4d04dbdf54ed76c5a4b06b9c5b36R1609